extra/utils/permission.js

```diff
@@ -0,0 +1,22 @@
+let permission = {
```

Comment:
@divyansh0777 What if I use _const_?

Reply:
```suggestion
let permission = {
```
@NishantNaithani1997 we can use **const** as well **instead** of let if we want to.